docs/genome_to_collection.md

```diff
@@ -0,0 +1,235 @@
+# From KBase Genome to Collection
+
+This guide outlines the process of creating and activating a new collection from KBase narrative with Genome workspace
+objects.
+
+## Step 1: NERSC Account Setup
+The commands provided in this document are intended for execution within the terminals of 
+[NERSC Perlmutter](https://docs.nersc.gov/systems/perlmutter/). 
+If you don't have a NERSC account yet, you can create one via [Iris](https://docs.nersc.gov/iris/).
+Before proceeding, please ensure that you have an active NERSC account and are logged in to Perlmutter.
+
+```commandline
+ssh perlmutter.nersc.gov
+```
+
+## Step 2: Prepare Source Data Files (FASTA)
+
+### If you already have a KBase narrative with target Genome objects
+If you already possess a KBase narrative containing the desired Genome objects, download the FASTA files associated with
+the Genome objects from the KBase workspace to NERSC. The following command facilitates this process:
+
+```commandline
+# update arguments as needed
+workspace_id=69739
+kbase_collection=ENIGMA
+source_verion=2023.10
+env=CI
+token_filepath=path/to/token_file.txt  # This is a single-line file with the KBase token 
+
+cd /global/cfs/cdirs/kbase/collections/collections
+PYTHONPATH=. python src/loaders/workspace_downloader/workspace_downloader.py \
+    --workspace_id $workspace_id \
+    --kbase_collection $kbase_collection \
+    --source_ver $source_verion \
+    --root_dir $root_dir \
+    --env $env \
+    --token_filepath $token_filepath \
+    --retrieve_sample 
```

Comment:
Will this error out if there are no or missing samples?

Reply:
error out. added comments regarding to it.

Reply:
:+1:

Reply:
:+1: